assets/collapsible-content.css

```diff
@@ -0,0 +1,104 @@
+.layout-section-container {
+  padding-bottom: 5rem;
+  padding-top: 5rem;
+}
+
+@media screen and (min-width: 750px) {
+  .layout-section-container {
+    padding-bottom: 7rem;
+    padding-top: 7rem;
+  }
+}
```

Comment:
I would probably combine some of these same media query blocks into the same block to save on some lines since they are pretty close to each other.  Can be done later.